src/qibolab/platform.py

```diff
@@ -16,6 +16,15 @@
 from qibolab.qubits import Qubit, QubitId, QubitPair
 
 
+@dataclass
+class PlatformSettings:
+    """Platform settings and default execution settings read from the runcard."""
+
+    nshots: int = 1024
+    sampling_rate: int = int(1e9)
+    relaxation_time: int = int(1e5)
```

Comment:
I'm not sure but the sampling rate should be instrument dependent (?)
I understand that for now this is just a placeholder, right?

Reply:
I would also add some docstrings under each attribute even if they are pretty clear.
For example you could add the unit in the docstrings at least for sampling rate and relaxation time.